src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts

```diff
@@ -216,6 +280,28 @@ class OZWNodeConfig extends LitElement {
     }
   }
 
+  private _configValueChanged(ev: CustomEvent) {
+    this._configData[(<OzwHaForm>ev.currentTarget)!.parameter] =
+      ev.detail.value[Object.keys(ev.detail.value)[0]];
```

Comment:
Make a copy so the form is rerendered
```suggestion
    this._configData = {...this._configData, (<OzwHaForm>ev.currentTarget)!.parameter: ev.detail.value[Object.keys(ev.detail.value)[0]]};
```